salt/client/mixins.py

```diff
@@ -429,13 +440,29 @@ def print_async_event(self, suffix, event):
         if suffix in ('new', ):
             return
 
+        # --out/output and outputer= are synonyms.
+        # --out/output wins if both are defined
+        # Default to the outputter in the event
+        outputter = event.get('outputter', None)
+        # --out and --output are gracked internally with 'output'
+        # self.opts.get('out') will resolve to None
+        opts_outputter = self.opts.get('output', None)
+
+        # Override with opts_outputter?
+        if opts_outputter is not None:
+            # Emit a warning if both are defined and the outputter is changing
+            if outputter is not None and outputter != opts_outputter:
+                warning = "Both outputter=%s and --out/output=%s are defined. %s will be used." % (outputter, opts_outputter, opts_outputter)
```

Comment:
Good call. @ckochenower, can you replace this with string formatting using .format()?